src/getViewForSwagger2.ts

```diff
@@ -54,48 +114,53 @@ var getViewForSwagger2 = function(opts){
         definitions: []
     };
 
-    var latestMethodVersion = {}; /* Maps method name => max version */
+    const latestMethodVersion: LatestMethodVersion = {}; /* Maps method name => max version */
+
+    function isParamter(__: any, m: string): __ is ReadonlyArray<any> {
+        return m.toLowerCase() === 'parameters';
+    }
 
-    _.forEach(swagger.paths, function(api, path){
-        var globalParams = [];
+    forEach(swagger.paths, function(api, path){
+        let globalParams: ReadonlyArray<any> = [];
         /**
          * @param {Object} op - meta data for the request
          * @param {string} m - HTTP method name - eg: 'get', 'post', 'put', 'delete'
```

Comment:
Lets use descriptive param names here and below for better readability

Reply:
This whole file needs some love, i basically did just the necessary stuff to make it run :)